src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -337,9 +343,10 @@ function create_cache(mesh::DGMultiMesh, equations, dg::DGMultiFluxDiff, RealT,
   rhs_local_threaded = [allocate_nested_array(uEltype, nvars, (num_quad_points_total,), dg)  for _ in 1:Threads.nthreads()]
 
   return (; md, Qrst_skew,
-            VhP, Ph, invJ = inv.(md.J),
+            VhP, Ph, invJ = inv.(md.J), invVDM = inv(rd.VDM),
```

Comment:
I don't really like this part... I had another option: store the factorization of VDM and define `ldiv_by!` similarly as `mul_by!`.

Reply:
I don't think this is a problem since it's just precomputed once. 

I'd suggest moving `invVDM` to the filtering struct type instead though.

Reply:
Fixed. Thanks!